app/daemon.go

```diff
@@ -84,11 +84,22 @@ type MenderShellDaemon struct {
 	homeDir                 string
 	shellsSpawned           uint
 	debug                   bool
+	router                  session.Router
 }
 
 func NewDaemon(config *configuration.MenderShellConfig) *MenderShellDaemon {
 	ctx, ctxCancel := context.WithCancel(context.Background())
 
+	routes := make(session.ProtoRoutes)
+	if config.FileTransfer.Enable {
```

Comment:
I'm not 100% sure, but I believe the discussion led us to enable file transfer by default, and provide an option to disable it. This way, it will work out-of-the-box without any special configuration (e.g. onboarding).

Reply:
Changed to `!config.FileTransfer.Diable`